cypress/e2e/5-ndv.cy.ts

```diff
@@ -0,0 +1,65 @@
+import { WorkflowsPage, WorkflowPage } from '../pages';
+import { v4 as uuid } from 'uuid';
+
+const workflowsPage = new WorkflowsPage();
+const workflowPage = new WorkflowPage();
+
+describe('NDV', () => {
+	beforeEach(() => {
+		cy.resetAll();
+		cy.skipSetup();
+		workflowsPage.actions.createWorkflowFromCard();
+		workflowPage.actions.renameWorkflow(uuid());
+		workflowPage.actions.saveWorkflowOnButtonClick();
+	});
+
+	it('should show up when double clicked on a node and close when Back to canvas clicked', () => {
+		workflowPage.actions.addInitialNodeToCanvas('Manual Trigger');
+		workflowPage.getters.canvasNodes().first().dblclick();
+		cy.getByTestId('ndv').should('be.visible');
+		cy.getByTestId('back-to-canvas').click()
+		cy.getByTestId('ndv').should('not.be.visible');
```

Comment:
Why not create NDV getters for these? We might want to reuse them beyond this suite and it's much easier to get them suggested by IDE than searching for the right ids

Reply:
created a page object